Opcodes/pvsops.cpp

```diff
@@ -448,7 +448,6 @@ struct TPrint : csnd::Plugin<0, 1> {
   PrintThread t;
 
   int init() {
-    csound->plugin_deinit(this);
```

Comment:
Maybe this commented out opcode should just be deleted?

Reply:
done